### PR TITLE
Add tests for Unicode v15 Identifier{Start,Part}

### PR DESCRIPTION
**Pull request by mathiasbynens**
Issue: https://github.com/tc39/ecma262/issues/2904

### CODE REVIEW

File: test/language/identifiers/start-unicode-15.0.0-escaped.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
...A}\u{3232B}\u{3232C}\u{3232D}\u{3232E}\u{3232F}\u{32330}\u{32331}\u{32332}\u{32333}\u{32334}\u{32335}\u{32336}\u{32337}\u{32338}\u{32339}\u{3233A}\u{3233B}\u{3233C}\u{3233D}\u{3233E}\u{3233F}\u{32340}\u{32341}\u{32342}\u{32343}\u{32344}\u{32345}\u{32346}\u{32347}\u{32348}\u{32349}\u{3234A}\u{3234B}\u{3234C}\u{3234D}\u{3234E}\u{3234F}\u{32350}\u{32351}\u{32352}\u{32353}\u{32354}\u{32355}\u{32356}\u{32357}\u{32358}\u{32359}\u{3235A}\u{3235B}\u{3235C}\u{3235D}\u{3235E}\u{3235F}\u{32360}\u{32361}\u{32362}\u{32363}\u{32364}\u{32365}\u{32366}\u{32367}\u{32368}\u{32369}\u{3236A}\u{3236B}\u{3236C}\u{3236D}\u{3236E}\u{3236F}\u{32370}\u{32371}\u{32372}\u{32373}\u{32374}\u{32375}\u{32376}\u{32377}\u{32378}\u{32379}\u{3237A}\u{3237B}\u{3237C}\u{3237D}\u{3237E}\u{3237F}\u{32380}\u{32381}\u{32382}\u{32383}\u{32384}\u{32385}\u{32386}\u{32387}\u{32388}\u{32389}\u{3238A}\u{3238B}\u{3238C}\u{3238D}\u{3238E}\u{3238F}\u{32390}\u{32391}\u{32392}\u{32393}\u{32394}\u{32395}\u{32396}\u{32397}\u{32398}\u{32399}\u{3239A}\u{3239B}\u{3239C}\u{3239D}\u{3239E}\u{3239F}\u{323A0}\u{323A1}\u{323A2}\u{323A3}\u{323A4}\u{323A5}\u{323A6}\u{323A7}\u{323A8}\u{323A9}\u{323AA}\u{323AB}\u{323AC}\u{323AD}\u{323AE}\u{323AF};
```
Comment (nicolo-ribaudo) [inline test/language/identifiers/start-unicode-15.0.0-escaped.js:16]: Isn't this test testing only that `\u{1123F}` is accepted as an identifier start character? This test would pass even if `\u{11240}` is only accepted as an identifier continuation character.

Comment (mathiasbynens) [reply test/language/identifiers/start-unicode-15.0.0-escaped.js]: Good point! Fixed in https://github.com/mathiasbynens/caniunicode/commit/49b9c03a63256b909bad5d1b01ce727b6473a0db and pushed to this PR.

**Review by Ms2ger — APPROVED** [review]
